api/reference/providers.adoc

```diff
@@ -77,13 +77,20 @@ The *type* attribute specifies the supported provider class names.
 This list primarily includes:
 
 [cols="<",width="30%"]
-|================
-| EmsRedhat
-| EmsVmware
-| EmsOpenstack
-| EmsKubernetes
-| etc.
-|================
+|===================================================
+| ManageIQ::Providers::Foreman::ConfigurationManager
+| ManageIQ::Providers::Vmware::InfraManager
+| ManageIQ::Providers::Microsoft::InfraManager
+| ManageIQ::Providers::Redhat::InfraManager
+| ManageIQ::Providers::Openstack::InfraManager
+| ManageIQ::Providers::Amazon::CloudManager
+| ManageIQ::Providers::Azure::CloudManager
+| ManageIQ::Providers::Openstack::CloudManager
+| ManageIQ::Providers::Atomic::ContainerManager
+| ManageIQ::Providers::Kubernetes::ContainerManager
+| ManageIQ::Providers::Openshift::ContainerManager
+| ManageIQ::Providers::Foreman::ProvisioningManagerEmsRedhat
```

Comment:
Please drop the 2 Foreman ones here, usually created/managed by the Foreman::Provider added below,
sort the list  since it grew and add back the etc. 

```
ManageIQ::Providers::Amazon::CloudManager
ManageIQ::Providers::Atomic::ContainerManager
ManageIQ::Providers::Azure::CloudManager
ManageIQ::Providers::Google::CloudManager
ManageIQ::Providers::Kubernetes::ContainerManager
ManageIQ::Providers::Microsoft::InfraManager
ManageIQ::Providers::Openshift::ContainerManager
ManageIQ::Providers::Openstack::CloudManager
ManageIQ::Providers::Openstack::InfraManager
ManageIQ::Providers::Redhat::InfraManager
ManageIQ::Providers::Vmware::InfraManager
etc.
```

Thanks.